schematics/playground-module/generate-missing-modules.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { basename, dirname, normalize, Path } from '@angular-devkit/core';
+import { apply, chain, DirEntry, filter, mergeWith, Rule, template, Tree, url } from '@angular-devkit/schematics';
+import {
+  generateCurrentDirImport,
+  generateFeatureModuleClassName,
+  generateFeatureModuleFileName,
+  generateRoutingModuleClassName,
+  generateRoutingModuleFileName,
+  getModuleDirs,
+  getPlaygroundRootDir,
+  hasComponentsInDir,
+  hasFeatureModuleInDir,
+  hasRoutingModuleInDir,
+  isFeatureModule,
+} from '../utils';
+
+export function generateMissingModules(tree: Tree): Rule {
+  const moduleDirs = [ getPlaygroundRootDir(tree), ...getModuleDirs(tree) ];
+
+  return chain(moduleDirs.map(moduleDir => fromTemplate(tree, optionsFromDir(moduleDir))));
```

Comment:
Let's add some comment description for this mapping. It was pretty hard to understand what's going on here 😄